src/deferred.js

```diff
@@ -304,6 +304,16 @@ jQuery.extend( {
 					// progress_callbacks.lock
 					tuples[ 0 ][ 2 ].lock
 				);
+
+				tuple[ 3 ].add(
```

Comment:
You mean adding it to the `list.add` above? I'll try it out... I don't recall why I did it here :/